05/index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="cuentos cortos o fabulas con moralejas">
+    <meta name="keyword" content="cuentos, cortos, moralejas, lectura, escritura, aprendizaje">
+    <title>Formularios y Cuentos</title>
+    <meta name="author" content="Jesus David Ceballos Parra">
+  </head>
+  <body>
+    <header>
+      <h1>LAS ZORRA Y LAS UVAS</h1>
+      <p>En pleno Otoño, Cuando las uvas mostraban suculentos ramos, Estaba una zorra tumbada debajo de un arból, Hechando un sueñecito.
+        Justo cuando se despertó, Se fijó en un ramo de uvos que va se veia maduro y le parecio un maravilloso bocado, que no debia dejar escapar.
+        Cuando se incorporó, pudo observar que no seria tarea facíl alcanzar las uvas, Pero no quizo rendirse tan pronto.
+        Comenzó a salta y a saltar, sin obtener ningún éxito.  Decidió acercar algúnas pierdas para elevarse, pero tampoco dio resultado.
+        tanto esfuerzo tenia a la zorra exahusta y, Finalmente, Decidió abandonar el racimo de uvas, pues le parecía imposible alcanzarlo.
+        Cuando ya se marchaba bastante frustada por no haber conseguidos sus uvas
+        Vio a un pajarillo junto al arból de cual colgaba el racimo.
+        La pequeña ave había presenciado toda la escena. La zorra, al darse cuenta de que la habían visto fracasar,se sintió avergonzada y quiso que el pajaro no pensara que no podia conseguir todo lo que ella quisiera, asi que dijo llena de orgullo:
+      </p>
+    </header>
+    <nav>
+      <h2>Dialogo de la zorra</h2>
+      <ol type="disc">
+        <ul>No es que no haya alcanza las uvas, lo habría hecho si merecieran la pena.</ul>
+        <ul>Sin embargo, a medida que me he ido acercando, he comprobar que las uvas no estaban maduras tal y como pensaba.</ul>
+        <ul>Yo tengo un paladar demasiado fino como para tomar uvas que no estén en perfecto estado de maduracíon.</ul>
+      </ol>
+      <p>Y así fue como la zorra se inventó una historia y se convenció a sí misma de que las uvas no eran merecedoras de su finísimo paladar</p>
+    </nav>
+    <section>
+      <h2>Moralejas de la zorra y las uvas</h2>
+      <ol>
+        <li>Cuando quieras conseguir algo, debes esforzarte hasta lograrlo.</li>
+        <li>No pongas excusas, si no consigues lo que te has propuesto.</li>
+        <li>Acepta la derrota.</li>
+        <li>Inténtalode nuevo y no te rindas.</li>
+      </ol>
+    </section>
+    <article>
+      <form>
+        <fieldset>
+          <legend>Preguntas</legend>
+          <label for="estación">¿En qué estación del año crecen las uvas?</label>
+          <select name="Seleccione unas estación" id="estación">
+            <option>Primavera</option>
+            <option>Verano</option>
+            <option>Otoño</option>
+            <option>Invierno</option>
```

Comment:
Some attribute is missed in option tag / Etiquetas option tienen un atributo faltante